pyproject.toml

```diff
@@ -12,7 +12,7 @@ classifiers = [
 ]
 dependencies = [
     "matplotlib",
-    "numpy",
+    "numpy<2.0.0",
```

Comment:
@odstrcilt can you please explain? Why should this be done across the whole package and for all users?

Reply:
This was the only option how to make the regression test pass. It has produced this error:
https://github.com/fsciortino/Aurora/actions/runs/10012842877/job/27679164044?pr=103
Enrico suggested using the older numpy 
https://github.com/fsciortino/Aurora/pull/99#issuecomment-2240023051